exp/api/v1beta1/azuremanagedmachinepool_types.go

```diff
@@ -165,23 +165,23 @@ type AzureManagedMachinePoolList struct {
 }
 
 // GetConditions returns the list of conditions for an AzureManagedMachinePool API object.
-func (m *AzureManagedMachinePool) GetConditions() clusterv1.Conditions {
-	return m.Status.Conditions
+func (r *AzureManagedMachinePool) GetConditions() clusterv1.Conditions {
```

Comment:
why "r" as a receiver here?

Reply:
For `AzureManagedMachinePool ` -- code base has `r` as well as `m` as receivers and the `stylecheck` linter complains about that as consistency is preferred. Semantically, `m` looks more appropriate to me as compared to `r`

Reply:
I opted for using `r` over `m` because it had more ocurrencies, but I agree thar `m` looks more appropriate, I will make the change.